components/pkg-export-docker/src/build.rs

```diff
@@ -24,6 +24,7 @@ use crate::{accounts::{EtcGroupEntry,
                     package::{PackageArchive,
                               PackageIdent,
                               PackageInstall},
+                    util::docker,
```

Comment:
On linux I get a warning that this is unused.
```rust
#[cfg(windows)]
use crate::hcore::util::docker;
```

Reply:
great suggestions. will fix!